app/controllers/settings/featured_tags_controller.rb

```diff
@@ -1,9 +1,6 @@
 # frozen_string_literal: true
 
 class Settings::FeaturedTagsController < Settings::BaseController
-  layout 'admin'
-
-  before_action :authenticate_user!
   before_action :set_featured_tags, only: :index
```

Comment:
Hm, wouldn't it be consistent to require the account not to be suspended, here?

Reply:
The controller inherits from Settings::BaseController (which sets layout and authenticate_user!), which inherits from ApplicationController (which does require_functional_account!). So, this controller is actually only accessible to funtional accounts. Suspended users excluded.

Reply:
I'll be honest my brain gets fried too trying to think of what pending, unconfirmed, suspended, disabled, memorialized users are supposed to be able to access. I thought uniting all of that into "functional" would simplify it, and so it was, up until we needed to customize some behaviours between them.

Reply:
oh ok, it does make sense here